instaloader/structures.py

```diff
@@ -876,6 +876,19 @@ def business_category_name(self) -> str:
     def biography(self) -> str:
         return self._metadata('biography')
 
+    @property
+    def entities(self) -> str:
```

Comment:
Could we make it return a List rather than a `str`? Would be more user-friendly.